doc/src/manual/interfaces.md

```diff
@@ -322,8 +344,8 @@ forms, in most case you only need to specialize the 3-argument form.) For this t
 that `SparseArray` is mutable (supports `setindex!`). [`similar()`](@ref) is also used to allocate result
 arrays for arithmetic on `AbstractArrays`, for instance:
 
-```julia
-julia> A + 4
+```jldoctest squarevectype
+julia> A .+= 4
```

Comment:
So, what method should be defined for `SparseArray` (or any user defined array type for that matter) for this example to work?